src/robotFactory.js

```diff
@@ -1,10 +1,63 @@
 'use strict';
 
-class BaseRobot {}
+class BaseRobot {
+  constructor(name, weight, coords, chipVersion) {
+    this.name = name;
+    this.weight = weight;
+    this.coords = coords;
+    this.coords.x = coords.x || 0;
+    this.coords.y = coords.y || 0;
+    this.chipVersion = chipVersion;
+  }
 
-class FlyingRobot {}
+  goForward(step = 1) {
+    this.coords.y += step;
+  }
+  goBack(step = 1) {
+    this.coords.y -= step;
+  }
+  goLeft(step = 1) {
+    this.coords.x -= step;
+  }
+  goRight(step = 1) {
+    this.coords.x += step;
+  }
+  getInfo() {
+    return `Robot: ${this.name}, Chip version: ${
+      this.chipVersion}, Weight: ${this.weight}`;
```

Comment:
I think it's better to use concat here. Cause this part looks not very nice
```suggestion
     return `Robot: ${this.name}, `
    + `Chip version: ${this.chipVersion}, `
    + `Weight: ${this.weight}`;
```

Reply:
Yeah, it was mostly because of linter error about too many chars in one line.